tests/unit/services.py

```diff
@@ -295,3 +298,8 @@ def user_service(mock_service):
 @pytest.fixture
 def vitalsource_service(mock_service):
     return mock_service(VitalSourceService)
+
+
+@pytest.fixture
+def grouping_plugin(mock_service):
+    return mock_service(GroupingServicePlugin)
```

Comment:
If this is used outside of the grouping service, this seems like a good move. Although with some of the changes suggested above, we might not need this.